controller.py

```diff
@@ -1,15 +1,17 @@
 import View
-import model
+from model import Core
 
 
-def callback1(user_name, u_pass):
-    x = model.login(user_name, u_pass)
+def callback1(user_name, user_pass):
+    core.add_user(user_name, user_pass)
```

Comment:
Why are you adding a new user each time one tries to login? 
Remove this line please.